### ISSUE TITLE
Chakram flight paths are buggy

### ISSUE THREAD

**Issue by elunna** [elunna/hackem#275]
Chakram flight paths are beyond buggy and can show as flying through walls.

Chakram sometimes have like a 3s delay on throws

**Comment by hatcatter** [comment]
Janis had a proper crash with one on the following ttyrec: https://hdf-eu.s3.amazonaws.com/ttyrec/p/parasite/hackem/ttyrec/2023-06-01.14:35:07.ttyrec.gz

**Comment by elunna** [comment]
Looking at the TTYREC, the last time inventory is displayed we see:
<img width="302" alt="image" src="https://github.com/elunna/hackem/assets/5000096/95676015-6997-4b03-ab98-5c841769bf72">

Player is two-weaponing kiku and snickersnee and has daggers in quiver.

**Comment by elunna** [comment]
I reverted a change with chakrams (4914db539) that I believe is most likely the culprit here. After this revert, I tried reproducing the problem by two-weaponing and having a stack of daggers quivered while I threw and recaught chakrams and had no issues.

Also tried throwing at a black dragon so one got disintegrated (if the problem was a destroyed chakram trying to return), but no issue.

I'll leave this one open for a bit for further testing.
